compiler/rustc_middle/src/middle/region.rs

```diff
@@ -358,6 +367,34 @@ impl ScopeTree {
         self.rvalue_scopes.insert(var, lifetime);
     }
 
+    pub fn record_local_access_scope(&mut self, var: hir::ItemLocalId, proposed_lifetime: Scope) {
+        debug!("record_local_access_scope(sub={:?}, sup={:?})", var, proposed_lifetime);
```

Comment:
Any reason not to have this as `var={:?}, proposed_lifetime={:?}` instead of `sub={:?}, sup={:?}`? I tend to find it tricky to remember what sub and sup are when I just see those two terms.

Reply:
Right, updated with your suggestion.

Reply:
Awesome, thanks!